### PR TITLE
fix: users cannot show the toolbar item when the editor is resized

### PR DESCRIPTION
**Pull request by js87zz**
### Please check if the PR fulfills these requirements
- [x] It's the right issue type on the title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has a description of the breaking change

### Description
* fixed that users cannot show the toolbar item when the editor is resized.
* related issue
  * #1674
  * #1687
  * #1738
* TODO: add e2e-test for ui test😬

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

**Review by jwlee1108 — APPROVED** [review]
리뷰 완료합니다.

File: apps/editor/src/ui/components/toolbar/toolbar.ts [hunk 1]
```diff
@@ -119,9 +123,10 @@ export class Toolbar extends Component<Props, State> {
   }
 
   addEvent() {
-    this.props.eventEmitter.listen('openPopup', this.openPopup);
+    const { eventEmitter } = this.props;
+
     this.handleResize = throttle(() => this.setState(this.classifyToolbarItems()), 200);
-    window.addEventListener('resize', this.handleResize);
+    eventEmitter.listen('openPopup', this.openPopup);
```
Comment (adhrinae) [inline apps/editor/src/ui/components/toolbar/toolbar.ts:129]: 요 `openPopup` 이벤트는 다른데서 일으키는거죠?

File: apps/editor/src/ui/components/toolbar/toolbar.ts [hunk 2]
```diff
@@ -231,6 +236,7 @@ export class Toolbar extends Component<Props, State> {
     // classify toolbar and dropdown toolbar after DOM has been rendered
     this.setState(this.classifyToolbarItems());
     this.appendTooltipToRoot();
+    this.resizeObserver.observe(this.refs.el);
```
Comment (adhrinae) [inline apps/editor/src/ui/components/toolbar/toolbar.ts:239]: [`beforeDestroy` 에서](https://github.com/nhn/tui.editor/blob/474413341b133df5d3adc9736a1e31c35a6451e4/apps/editor/src/ui/components/toolbar/toolbar.ts#L260-L263) 옵저버를 제거해주어야 하지 않을까요?

**Review by adhrinae — COMMENTED** [review]
리뷰 완료합니다.

File: apps/editor/src/ui/components/toolbar/toolbar.ts (hunk 2 above)
Comment (js87zz) [reply apps/editor/src/ui/components/toolbar/toolbar.ts:239]: 아 넵 깜빡했네요 감사합니다!ㅎㅎ

File: apps/editor/src/ui/components/toolbar/toolbar.ts (hunk 1 above)
Comment (js87zz) [reply apps/editor/src/ui/components/toolbar/toolbar.ts:129]: 넵 이거는 툴바 버튼 눌렀을 때 발생합니다. 에디터 UI에 별도 상태관리 메커니즘은 과하다고 생각이되서 간편하게 이벤트 버스로 통신하고 있습니다.ㅎㅎ

**Review by lja1018 — APPROVED** [review]
리뷰완료합니다.

**Pull request merged**